artifacts/deploy/kube-controller-manager.yaml

```diff
@@ -42,7 +42,7 @@ spec:
             - --cluster-name=karmada
             - --cluster-signing-cert-file=/etc/karmada/pki/ca.crt
             - --cluster-signing-key-file=/etc/karmada/pki/ca.key
-            - --controllers=namespace,garbagecollector,serviceaccount-token,ttl-after-finished
+            - --controllers=namespace,garbagecollector,serviceaccount-token,ttl-after-finished,csrapproving,csrcleaner,csrsigning
```

Comment:
It is no problem to only support `csr` now, and I add a document about `car` at https://github.com/karmada-io/website/pull/142.

Perhaps we can enable `bootstrapsigner` and `tokencleaer` as well in the  future. Like https://github.com/karmada-io/karmada/blob/23ec2afa8b4e09edcf0c7352b11bd477eab7593d/pkg/karmadactl/cmdinit/kubernetes/deployments.go#L270

But currently, only karmada control palne that is created by `karmadactl init` can use `karmadactl register` command.

Reply:
It would be better if the cluster configuration installed in the karamda helm chart mode and the cluster configuration installed in the karmadactl mode are consistent

Reply:
It quite makes sense. Now the community has several different ways. If we need to add new features in the future, we may need to modify the yaml in multiple places, which greatly increases our maintenance cost and also leads to the fork of each version. We may need to develop a good strategy for maintaining source files for different installations.